### ISSUE TITLE
FeathersNumberInput a11y support is incomplete

### ISSUE THREAD

**Issue by viridia** [bevyengine/bevy#25243]
## What problem does this solve or what need does it fill?

Currently we only set the role (which is wrong) and the `disabled` attribute.

The reason the role is wrong is because we are using the role from `FeathersTextInput`, but our actual role should be `RangeInput`.

Also we need to set the property representing the current value. This will be tricky because `TextInput` is also going to want to set a value, which is a string.

## What solution would you like?

It would probably be better to do this _after_ #25213 - then we can add support for all the a11y properties.
